tests/integration/systemsstate/test_systemsstate.py

```diff
@@ -0,0 +1,242 @@
+from typing import List
+
+import pytest
+from nisystemlink.clients.core._http_configuration import HttpConfiguration
+from nisystemlink.clients.systemsstate import models
+from nisystemlink.clients.systemsstate import SystemsStateClient
+
+
+@pytest.fixture(scope="class")
+def client(enterprise_config: HttpConfiguration) -> SystemsStateClient:
+    """Fixture to create a SystemsStateClient instance."""
+    return SystemsStateClient(enterprise_config)
+
+
+@pytest.fixture
+def default_work_space() -> str:
+    """Fixture that returns the default workspace id"""
+    return "846e294a-a007-47ac-9fc2-fac07eab240e"
```

Comment:
Tests should be agnostic of the SLE instance and equipped to work with a fresh deployment by taking care of creating/deleting any necessary entities or mock them in the requests response.
```suggestion
    return ""
```